sections/featured-product.liquid

```diff
@@ -292,6 +298,62 @@
                       </span>
                     {%- endif -%}
                   </div>
+                  {%- if product.quantity_price_breaks_configured? -%}
+                    <volume-pricing class="parent-display" id="Volume-{{ section.id }}">
+                      {%- if product.selected_or_first_available_variant.quantity_price_breaks.size > 0 -%}
+                        <span class="caption-large">{{ 'products.product.volume_pricing.title' | t }}</span>
+                        <ul class="list-unstyled no-js-hidden">
+                          <li>
+                            <span>{{ product.selected_or_first_available_variant.quantity_rule.min }}+</span>
```

Comment:
Let's remember to use locales in the few places that need it

Reply:
I have an issue to ensure we do :)